### PR TITLE
Add Reapchain Mainnet

### PR DESCRIPTION
**Pull request by helloreap**
Please review to merge it.

### CODE REVIEW

File: cosmos/reapchain_221230.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/reapchain/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "REAP",
+    "coinMinimalDenom": "areap",
+    "coinDecimals": 18,
+    "coinGeckoId": "reapchain",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/reapchain/areap.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/reapchain",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/reapchain",
```
Comment (HeesungB) [inline cosmos/reapchain_221230.json:15]: @helloreap Could you remove `walletUrl` and `walletUrlForStaking` field?
It is invalid url.

**Comment by HeesungB** [comment]
@helloreap I think that your chain id is `reapchain_2022-1`. Please check your chain id.
```
RPC endpoint has different chain id (expected: reapchain_221230-1, actual: reapchain_2022-1)
```

**Comment by helloreap** [comment]
@HeesungB We issued a PR that the mainnet will be restarted with a new chainid. Should I close the PR and do it again?

**Comment by HeesungB** [comment]
@helloreap 
Leave it as it is.
Launch the mainnet and tell me.

**Comment by helloreap** [comment]
@HeesungB 
How are you?
Today we officially opened our mainnet.
Please register on the keplr chain list.

**Comment by helloreap** [comment]
@HeesungB 
We'll issue a new pull request to resolve the above issue.

**Pull request closed without merging**